source/guide_wsgidav.rst

```diff
@@ -0,0 +1,200 @@
+.. highlight:: console
+
+.. author:: Sebastian Burschel <https://github.com/SeBaBu/>
+
+.. tag:: web
+.. tag:: file-storage
+.. tag:: webdav
+
+.. sidebar:: About
+
+  .. image:: _static/images/wsgidav.png
+      :align: center
+
+############
+WsgiDAV
+############
+
+.. tag_list::
+
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+MIT license.
+
+  * https://github.com/mar10/wsgidav/blob/master/LICENSE
+
+
+Installation
+============
+
+Use pip3 to install wsgidav:
+
+::
+
+ [isabell@stardust ~]$ pip3 install wsgidav cheroot --user
+
+Create the data folder:
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/webdav
+
+
+Add additional folders you want to access via webdav with:
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/webdav/<newfoldername>
+
+config file
+======================
+
+.. note::
+
+    Only use spaces to indent in the config file! No tabs!
+
+
+The following template is the minimum to get wsgidav up and running. You have to change provider_mapping, user_mapping and the port (if another service is already running on 8080).
+If you miss some functionality or want more information, take a look at the sample file and the explanation of the configuration options:
+https://wsgidav.readthedocs.io/en/latest/user_guide_configure.html#sample-wsgidav-yaml
+
+
+Config file path:
+
+::
+
+ ~/.wsgidav/wsgidav.yaml
```

Comment:
Since `~/.wsgidav` isn't in the config search path of wsgidav anyway, what do you think about putting the config file in the already present `~/etc`?